package/tests/test_api/example_pipelines.json

```diff
@@ -129,18 +119,18 @@
   ],
   "layers": ["raw", "model_inputs"],
   "tags": [
-    { "id": "split", "name": "Split" },
-    { "id": "train", "name": "Train" }
+    { "id": "split", "name": "split" },
+    { "id": "train", "name": "train" }
   ],
   "pipelines": [
-    { "id": "__default__", "name": "Default" },
-    { "id": "data_science", "name": "Data Science" },
-    { "id": "data_processing", "name": "Data Processing" }
+    { "id": "__default__", "name": "__default__" },
```

Comment:
nit - maybe for consistency we can replace full_name to name everywhere since we just only send 'name' now.